cinema/views.py

```diff
@@ -56,3 +116,27 @@
             return MovieSessionDetailSerializer
 
         return MovieSessionSerializer
+
+
+class OrderPagination(PageNumberPagination):
+    page_size = 3
+    page_size_query_param = "page_size"
+    max_page_size = 3
+
+
+class OrderViewSet(viewsets.ModelViewSet):
+    queryset = Order.objects.all()
+    serializer_class = OrderSerializer
+    pagination_class = OrderPagination
+
+    def get_queryset(self):
+        queryset = self.queryset.prefetch_related("tickets__movie_session")
+        return queryset.filter(user=self.request.user)
```

Comment:
Clean Code: You've added a function to check your solution here, which is a good practice. However, don't forget to delete them when you are ready to commit and push your code.